### ISSUE TITLE
Using ODROID XU4 with Ubuntu 16 and ROS Kinetic

### ISSUE THREAD

**Issue by mzahana** [uzh-rpg/rpg_svo_example#25]
Hi.

Does SVO 2.0 work with ODROID XU4 with Ubuntu 16 and ROS Kinetic? If yes, are the provided binaries enough ? Are there any extra/different steps that need to be done?

Thanks.

**Comment by nightpool** [comment]
No, it seems that the ARM build code assumes ROS Indigo and vice versa (the non-ARM code assumes 16.04 and ROS Kinetic) See #21

**Comment by mzahana** [comment]
Thanks. Does the package work for you as per instructions or you needed extra steps?

**Comment by nightpool** [comment]
It works as long as you're using Ubuntu 14.04 and ROS indigo

**Comment by FPSychotic** [comment]
I had that and error 2 when compile in imx6 with armv7a, I gonna try with fresh install in SSD and swap memory

**Comment by FPSychotic** [comment]
I had that and error 2 when compile in imx6 with armv7a, I gonna try with fresh install in SSD and swap memory
